### DISCUSSION TITLE
After upgrading Chia to version 2.01, the cooperative became empty

### DISCUSSION THREAD

**Discussion by zsvvv** [Chia-Network/chia-blockchain#16497] [Support]
I am a self-farmer, and I have nowhere to claim the newly mined 1.75 rewards. What should I do

**Comment by BrandtH22** [comment]
Hey @zsvvv , it sounds like your plotNFT might not be showing up properly.

Depending on the version you have upgraded from you might need to have a new config file created and/or resync the wallet.
Steps for each are below:

Having a new config created:
```
1. Stop the chia client
2. Rename the current config.yaml to config.yaml.old `~\.chia\mainnet\config\`
3. Run `chia init`
4. (OPT) Copy any custom settings from config.yaml.old to the new config.yaml (drive and trusted peer)
5. (OPT) Start the chia client
```

Resyncing the wallet:
```
1. Shut down chia
2. Delete the wallet db `~\.chia\mainnet\wallet\db\` (or rename it for archival purposes)
3. Restart chia and let it sync fully
```

**Reply by zsvvv** [reply]
thanks!

**Reply by BrandtH22** [reply]
Most certainly! 
Did that solve the issue?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
